examples/experimental/dagster-airlift/examples/dbt-example/dbt_example/dagster_defs/complete.py

```diff
@@ -7,23 +7,30 @@
     lakehouse_existence_check_defs,
     specs_from_lakehouse,
 )
+from dbt_example.dagster_defs.utils import apply_eager_automation
 from dbt_example.shared.load_iris import CSV_PATH, DB_PATH, IRIS_COLUMNS
 
 from .constants import dbt_manifest_path, dbt_project_path
 
 defs = Definitions.merge(
     defs_from_lakehouse(
-        specs=specs_from_lakehouse(csv_path=CSV_PATH),
+        specs=specs_from_lakehouse(
+            csv_path=CSV_PATH, automation_condition=AutomationCondition.on_cron("0 0 * * *")
+        ),
         csv_path=CSV_PATH,
         duckdb_path=DB_PATH,
         columns=IRIS_COLUMNS,
     ),
-    dbt_defs(
-        manifest=dbt_manifest_path(),
-        project=DbtProject(dbt_project_path()),
+    apply_eager_automation(
+        dbt_defs(
+            manifest=dbt_manifest_path(),
+            project=DbtProject(dbt_project_path()),
+        )
     ),
-    lakehouse_existence_check_defs(
-        csv_path=CSV_PATH,
-        duckdb_path=DB_PATH,
+    apply_eager_automation(
+        lakehouse_existence_check_defs(
+            csv_path=CSV_PATH,
+            duckdb_path=DB_PATH,
+        ),
```

Comment:
Until we could do this with a one-liner I think it should use a traditional schedule. cc: @OwenKephart for guidance here.

Reply:
Just put up: https://github.com/dagster-io/dagster/pull/24613, which enables setting a default condition on an AutomationConditionSensorDefinition: https://github.com/dagster-io/dagster/pull/24613/files#diff-e24091c0e2d196b1261e2510fb16c2ce7153fad761fb91d843403cc5c0a3e2c7R18